tdrs-backend/tdpservice/settings/local.py

```diff
@@ -22,6 +22,3 @@ class Local(Common):
     EMAIL_HOST = "localhost"
     EMAIL_PORT = 1025
     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
-
-    # Session Timeout
-    SESSION_TIMEOUT = 5
```

Comment:
Removing this sets the local `SESSION_TIMEOUT` to the setting in `common.py` which is 30